web/app.js

```diff
@@ -205,6 +206,14 @@ const PDFViewerApplication = {
       if (mode) {
         document.documentElement.classList.add(mode);
       }
+      // if (typeof PDFJSDev === "undefined" || PDFJSDev.test("TESTING")) {
```

Comment:
Tracking comment to make sure the commented out code here and below gets replaced before merging.